infra/terraform/hash/hash_application/main.tf

```
@@ -418,6 +430,13 @@ resource "aws_security_group" "app_sg" {
     description = "Allow connections AWS SES"
     cidr_blocks = [var.vpc.cidr_block]
   }
+  egress {
+    from_port   = var.temporal_port
+    to_port     = var.temporal_port
+    protocol    = "tcp"
+    description = "Allow outbound GRPC connections to Temporal"
+    cidr_blocks = ["0.0.0.0/0"]
```

isn't Temporal in the same VPC?
```suggestion
    cidr_blocks = [var.vpc.cidr_block]
```